From a.beregalov@gmail.com  Tue Apr 21 16:13:43 2009
From: Alexander Beregalov <a.beregalov@gmail.com>
Date: Fri, 17 Apr 2009 15:19:14 +0400
Subject: USB: musb: fix build when !CONFIG_PM
To: gregkh@suse.de
Message-ID: <20090417111914.GA31310@orion>
Content-Disposition: inline


Fix this build error when CONFIG_PM is not set:
drivers/usb/musb/musb_core.c:2232: error: 'musb_resume_early' undeclared here

Signed-off-by: Alexander Beregalov <a.beregalov@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/musb_core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/musb/musb_core.c
+++ b/drivers/usb/musb/musb_core.c
@@ -2191,7 +2191,7 @@ static int musb_resume_early(struct plat
 
 #else
 #define	musb_suspend	NULL
-#define	musb_resume	NULL
+#define	musb_resume_early	NULL
 #endif
 
 static struct platform_driver musb_driver = {
